### PR TITLE
Reuse data model from the server

### PR DESCRIPTION
**Pull request by H34D**

### CODE REVIEW

File: src/app/shared/ethstats.service.ts [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 import { environment } from 'src/environments/environment'
 
-import { EthstatsNode, EthstatsCharts } from './store/ethstats'
+import { EthstatsCharts, EthstatsNode } from './store/ethstats'
+import { Events } from '@celo/celostats-server/src/server/server/Events'
 
 // todo: Sebastian: i tried a long time removing this and i failed. Please make it more obvious
 export interface EthstatsServiceDataNode {
-  action: 'init' | 'add' | 'block' | 'pending' | 'stats'
+  action: Events.Init | Events.Add | Events.Block | Events.Pending | Events.Stats
```
Comment (Pedro-vk) [inline src/app/shared/ethstats.service.ts]: If Events is a enum, probably is enught using `actions: Events`, but if we just want to use some of them, is ok

Comment (Pedro-vk) [reply src/app/shared/ethstats.service.ts]: I've seen the next interface, makes sense to differentiate it :+1:

File: src/app/shared/ethstats.service.ts [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
   data: Partial<EthstatsNode>
 }
 
 // todo: Sebastian: i tried a long time removing this and i failed. Please make it more obvious
```
Comment (Pedro-vk) [inline src/app/shared/ethstats.service.ts]: Should we remove the comments?

Comment (H34D) [reply src/app/shared/ethstats.service.ts]: removed, thanks for the review.

**Pull request merged**